lib/cobra_commander/dependencies.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative "dependencies/yarn"
+require_relative "dependencies/bundler"
+require_relative "dependencies/multi_source"
+
+module CobraCommander
+  module Dependencies
+    def self.context(root, components, yarn: false, bundler: false)
+      sources = []
+      sources << Yarn.build(root) unless bundler
+      sources << Bundler.build(root, components) unless yarn
+      MultiSource.new(sources)
+    end
```

Comment:
Surely you mean this?

```suggestion
    def self.context(root, components, yarn: true, bundler: true)
      sources = []
      sources << Yarn.build(root) if bundler
      sources << Bundler.build(root, components) if yarn
      MultiSource.new(sources)
    end
```

Reply:
Also, what about this?

```suggestion
    def self.context(root, components_directory = "components", yarn: false, bundler: false)
      sources = []
      sources << Yarn.build(root) unless bundler
      sources << Bundler.build(root, components_directory) unless yarn
      MultiSource.new(sources)
    end
```

I don't see alternative values for `components_directory` being used very often.